src/Main.elm

```diff
@@ -31,8 +31,8 @@ type PageState
 
 type alias AuthedPageState =
     { user : User.User
-    , messageIds : RemoteData.WebData MessageId.Envelope
-    , threads : RemoteData.WebData Thread.Envelope
+    , messageIds : RemoteData.RemoteData Http.Error MessageId.Envelope
+    , threads : RemoteData.RemoteData Http.Error Thread.Envelope
```

Comment:
`RemoteData.WebData Model` is same as `RemoteData.RemoteData Http.Error Model`. So it's convenience alias since most of the time you are using `RemoteData` for http requests.